rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -501,13 +488,15 @@ def _process_csv_airdrop(
     # temporarily store this protocol's data here
     temp_found_data: dict[ChecksumEvmAddress, dict] = defaultdict(lambda: defaultdict(dict))
     temp_airdrop_tuples = []
-    for row in get_airdrop_data(airdrop_data, protocol_name, data_dir):
-        if len(row) < 2:
-            msg_aggregator.add_warning(f'Skipping airdrop CSV for {protocol_name} because it contains an invalid row: {row}')  # noqa: E501
-            break
-        addr, amount, *_ = row
-        # not doing to_checksum_address() here since the file addresses are checksummed
-        # and doing to_checksum_address() so many times hits performance
+    airdrop_lazy_df = get_airdrop_data(airdrop_data, protocol_name, data_dir)
```

Comment:
better name. I don't know what a df is

```suggestion
    airdrop_lazy_df_make_no_sense = get_airdrop_data(airdrop_data, protocol_name, data_dir)
```

Reply:
dataframe. https://docs.pola.rs/api/python/stable/reference/dataframe/index.html 
![imagen](https://github.com/rotki/rotki/assets/5068010/dd5fd1e6-d191-413d-a89b-8eda9dcd42b1)